rhel/SPECS/dbaas-controller.spec

```diff
@@ -0,0 +1,57 @@
+%undefine _missing_build_ids_terminate_build
+%global _dwz_low_mem_die_limit 0
+
+# do not strip debug symbols
+%global debug_package     %{nil}
+
+# commit value is dynamic, see build script:
+# https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm
+%global commit            0000000000000000000000000000000000000000
+%global shortcommit       %(c=%{commit}; echo ${c:0:7})
+%define build_timestamp   %(date -u +"%y%m%d%H%M")
+%define release           1
+%define rpm_release       %{release}.%{build_timestamp}.%{shortcommit}%{?dist}
+
+%global install_golang    0
+
+Name:       dbaas-controller
+Version:    %{version}
+Release:    %{rpm_release}
+Summary:    Simplified API for managing Percona Kubernetes Operators
+
+License:    AGPLv3
+URL:        https://github.com/percona-platform/dbaas-controller
+Source0:    https://github.com/percona-platform/dbaas-controller/archive/%{commit}/dbaas-controller-%{shortcommit}.tar.gz
+
+%if %{install_golang}
+BuildRequires:   golang >= 1.14.0
+%endif
+
+%description
+dbaas-controller exposes a simplified API for managing Percona Kubernetes Operators
+See the PMM docs for more information.
+
+
+%prep
+%setup -q -n dbaas-controller-%{commit}
+
+
+%build
+make release
```

Comment:
Please add

```
export PMM_RELEASE_VERSION=%{full_pmm_version}
export PMM_RELEASE_FULLCOMMIT=%{commit}
export PMM_RELEASE_BRANCH=""
```

`full_pmm_version` should be defined too, and is sed'ed too

Reply:
Added env vars and `full_pmm_version` spec variable